VSDiagnostics/VSDiagnostics/VSDiagnostics/Diagnostics/Strings/StringDotFormatWithDifferentAmountOfArguments/StringDotFormatWithDifferentAmountOfArgumentsAnalyzer.cs

```diff
@@ -37,28 +39,62 @@ private void AnalyzeNode(SyntaxNodeAnalysisContext context)
             var invokedMethod = context.SemanticModel.GetSymbolInfo(invocation);
```

Comment:
is `invokedMethod` even used anywhere except the line below? Because if it's not, you can just remove this intermediate variable